scripts/release-process/util.js

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
These files in `release-process` are not being used anywhere so are just hanging around so you know that means we can delete it :D